package.json

```diff
@@ -37,21 +37,21 @@
     "cf-core": "4.10.0",
     "cf-layout": "5.0.0",
     "cf-typography": "5.0.1",
-    "highcharts": "https://github.com/cfpb/highcharts-dist#fix-missing-plotline-label",
+    "highcharts": "6.2.0",
```

Comment:
☝️ This change is causing the geo map styling problems.

With the previous `#fix-missing-plotline-label` version of highcharts, the geo map looks like this:

![screen shot 2018-11-01 at 6 53 34 pm](https://user-images.githubusercontent.com/1060248/47884528-80181780-de07-11e8-82ce-74faa94479f6.png)

Inspecting unincorporated Montana shows that Highcharts applies an inline style to fill the svg path with the applicable light blue color:

`<path *shape and classes and other stuff* style="fill:#7eb7e8;vector-effect:non-scaling-stroke;"></path>`

With `6.2.0`, the geo map looks like this:

![screen shot 2018-11-01 at 5 31 11 pm](https://user-images.githubusercontent.com/1060248/47884327-ae492780-de06-11e8-8541-9d2a0416e20b.png)

The colors are off and inspecting unincorporated Montana shows:

`<path *path and classes and other stuff* style="stroke-width:inherit;"></path>`

The fork we were previously using was probably modified to fit our needs. Testing this PR with highcharts 5.0.14 (the latest version of 5.x) does not solve the problem. @cfarm do you remember why [we moved to the fork](https://github.com/cfpb/cfpb-chart-builder/commit/aeec92b488686d8432c5ff2f0f19de0a8b468849)?